crates/ark-contracts/arkchain/src/order/types.cairo

```diff
@@ -154,14 +155,28 @@ impl Felt252TryIntoOrderStatus of TryInto<felt252, OrderStatus> {
 struct ExecutionInfo {
     // The hash of the order to execute.
     order_hash: felt252,
+    // The hash of the offer order to be executed with an auction.
+    offer_hash: Option<felt252>,
```

Comment:
This hash may be used for other purposes than auction.
I would suggest to use something more generic like `reference_order_hash` or `linked_order_hash` or `related_order_hash` something like this. WDYT?

Reply:
related_order_hash